setup.py

```diff
@@ -32,6 +32,14 @@ def _get_long_description():
     author_email="engineering@asidatascience.com",
     url="https://github.com/ASIDataScience/dash-bootstrap-components",
     packages=find_packages(),
-    install_requires=["dash>=0.32.1"],
+    install_requires=["dash>=0.32.1", "dash-html-components"],
     include_package_data=True,
+    classifiers=[
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+    ],
+    extras_require={"Table.from_dataframe": ["numpy", "pandas"]},
```

Comment:
I wonder if the feature name should be something like `pandas-integration` or `pandas` or `dataframes`? We might imagine other functionality that maps from dataframes to components later?

Reply:
Yes, I was going to ask you about this actually as I thought my choice was a little clunky. I think `pandas` is nice and simple, but descriptive. I'll change to that.